ci/tests/puppeteer/scenarios/db-postgres-load/script.js

```diff
@@ -0,0 +1,16 @@
+const startPuppeteerLoadTest = require('puppeteer-loadtest');
+
+const file = "./ci/tests/puppeteer/scenarios/db-postgres-load/test.js";
```

Comment:
You should get the test script from the config object instead and use env variables for scenario name, etc.  See the x509-login scenario to see how to pass additional data into the script from the json config.

Reply:
```js
    let args = process.argv.slice(2);
    let config = JSON.parse(fs.readFileSync(args[0]));
    assert(config != null)

    console.log(`Certificate file: ${config.myTestFile}`);
```

Reply:
I moved the config parameters to the script.json. You can't really use the ${PWD} or ${SCENARIO} since those substitutions are done in the run.sh, and they aren't used in the x509-login. The javascript is generic enough that it could be moved into a method in cas.js, and I could probably do that, or I can leave it for whoever adds the second test that uses the same code (if that happens).